beetmoverscript/utils.py

```diff
@@ -51,12 +51,12 @@ def write_file(path, contents):
         fh.write(contents)
 
 
-def generate_beetmover_template_args(task, release_props):
+def generate_beetmover_template_args(task, release_props, cert, action):
     tmpl_key_platform = TEMPLATE_KEY_PLATFORMS[release_props["stage_platform"]]
```

Comment:
This `TEMPLATE_KEY_PLATFORMS` is solely used here to toggle between the template files (`fennec_nightly` vs `fennecx86_nightly`). But looking at the two templates for the two different Android platforms, I noticed they only differ by "arm" vs "i386 in labels. I could easily merge them and just toggle that part via a template arg. That way we get rid both of this env dict, the `fennecx86_*` yml files and the logic from this function. What do you think?